packages/@okta/vuepress-site/docs/guides/step-up-authentication/main/index.md

```diff
@@ -121,17 +123,18 @@ To check the returned ID token payload, copy the values and paste them into any
 {
   "sub": "00u47ijy7sRLaeSdC0g7",
   "ver": 1,
-  "iss": "https://{yourOktadomain}/oauth2/default",
+  "iss": "https://sharper.trexcloud.com/oauth2/default",
```

Comment:
`{yourOktadomain}`

Reply:
woops, forgot. thanks. I'll do that right now.